custom/icds/templates/partials/tableau_js.html

```diff
@@ -0,0 +1,213 @@
+<script>
+    var viz = {};
+    var workbook = {};
+    function initializeViz() {
+        var placeholderDiv = document.getElementById("tableauPlaceholder");
+        var url = "https://icds.commcarehq.org/#/views/{{ report_workbook }}/{{ report_worksheet }}";
+        var options = {
+            width: placeholderDiv.offsetWidth,
+            height: placeholderDiv.offsetHeight,
+            hideTabs: true,
+            hideToolbar: true,
+            onFirstInteractive: function () {
+                setUpNav(viz)
+            }
+        };
+        viz = new tableau.Viz(placeholderDiv, url, options);
+        // I am envisioning the history object possibly containing things like the base params for filtering
+        var historyObject = {
+            'location': '{{ report_worksheet }}'
+        };
+        history.pushState(historyObject, '', '{{ report_worksheet }}')
+    }
+
+    function setUpNav(viz) {
+        workbook = viz.getWorkbook();
+        var sheets = workbook.getPublishedSheetsInfo();
+        _.each(sheets, function(sheet) {
+            // TODO: This is a bit hacky, $0 was the only item in the sheet object though (maybe has to do with dashboards vs sheets?)
+            addNavigationLink(sheet.$0.name, sheet.$0.isActive)
+        });
+
+        // Apply inital params for user level filtering
+        var params = {
+            'user_view_by': '{{ user_view_by }}',
+            'user_state': '{{ user_state|default:'' }}',
```

Comment:
The failure mode here is to show everything, adding a country level should fix this.